engines/bops_api/schemas/odp/v0.6.0/shared/definitions.json

```diff
@@ -58,6 +58,10 @@
             "endDate": {
               "type": ["string", "null"],
               "format": "date"
+            },
+            "publicUrl": {
+              "type": ["string", "null"],
+              "format": "string"
```

Comment:
How about you’re moving this change to latest version 0.7.0?

Reply:
Yeah good catch.

@HonTastic2 We've just recently updated the schema version so we'll need to rebase this PR and add this to the 0.7.0 file before you can merge

Reply:
done